src/ucp/core/ucp_context.c

```diff
@@ -81,16 +81,19 @@ static ucs_config_field_t ucp_config_table[] = {
 
   {"TLS", UCP_RSC_CONFIG_ALL,
    "Comma-separated list of transports to use. The order is not meaningful.\n"
-   " - all    : use all the available transports.\n"
-   " - sm/shm : all shared memory transports.\n"
-   " - mm     : shared memory transports - only memory mappers.\n"
-   " - ugni   : ugni_rdma and ugni_udt.\n"
-   " - ib     : all infiniband transports.\n"
-   " - rc     : rc verbs (uses ud for bootstrap).\n"
-   " - rc_x   : rc with accelerated verbs (uses ud_x for bootstrap).\n"
-   " - ud     : ud verbs.\n"
-   " - ud_x   : ud with accelerated verbs.\n"
-   " - dc_x   : dc with accelerated verbs.\n"
+   " - all     : use all the available transports.\n"
+   " - sm/shm  : all shared memory transports.\n"
+   " - mm      : shared memory transports - only memory mappers.\n"
+   " - ugni    : ugni_rdma and ugni_udt.\n"
+   " - ib      : all infiniband transports.\n"
+   " - rc_v    : rc verbs (uses ud for bootstrap).\n"
+   " - rc_x    : rc with accelerated verbs (uses ud_x for bootstrap).\n"
+   " - rc      : rc_v and rc_x (preferably if available).\n"
+   " - ud_v    : ud verbs.\n"
+   " - ud_x    : ud with accelerated verbs.\n"
+   " - ud      : ud_v and ud_x (preferably if available).\n"
+   " - dc/dc_x : dc with accelerated verbs.\n"
+   " - tcp     : sockets over TCP/IP.\n"
```

Comment:
Every single time when I have to set one of these I go and review the aliases:
```
static ucp_tl_alias_t ucp_tl_aliases[] = {
  { "sm",    { "mm", "knem", "cma", "rdmacm", "sockcm", NULL } },
  { "shm",   { "mm", "knem", "cma", "rdmacm", "sockcm", NULL } },
  { "ib",    { "rc", "ud", "rc_mlx5", "ud_mlx5", "dc_mlx5", "rdmacm", NULL } },
  { "ud_v",  { "ud", "rdmacm", NULL } },
  { "ud_x",  { "ud_mlx5", "rdmacm", NULL } },
  { "ud",    { "ud_mlx5", "ud", "rdmacm", NULL } },
  { "rc_v",  { "rc", "ud:aux", "rdmacm", NULL } },
  { "rc_x",  { "rc_mlx5", "ud_mlx5:aux", "rdmacm", NULL } },
  { "rc",    { "rc_mlx5", "ud_mlx5:aux", "rc", "ud:aux", "rdmacm", NULL } },
  { "dc",    { "dc_mlx5", "rdmacm", NULL } },
  { "dc_x",  { "dc_mlx5", "rdmacm", NULL } },
  { "ugni",  { "ugni_smsg", "ugni_udt:aux", "ugni_rdma", NULL } },
  { NULL }
};
```

For each one of these I would list TLS that are used, otherwise its s very cryptic description,

Reply:
@shamisp added more descriptive information about aliases
could you review pls? hope it is what I meant

Reply:
updated `sm`/`shm`, `ugni`, `ib` and fixed `rc_x`

Reply:
@dmitrygx Much better, thanks